app/models/manageiq/providers/redhat/inventory/persister/infra_manager.rb

```diff
@@ -52,6 +45,13 @@ def initialize_inventory_collections
         :complete                     => !targeted?
       )
     end
+    add_collection(infra, :parent_blue_folders)
```

Comment:
Another minor cleanup item, you don't need to add the collection here and also [here](https://github.com/ManageIQ/manageiq-providers-ovirt/pull/480/files#diff-85616c128d7cc811ee4dc4c3225588d9R114)

Reply:
Absolutely, I'll fix it in the cleanup PR